src/main/java/graphql/analysis/QueryVisitorFragmentEnvironment.java

```diff
@@ -0,0 +1,13 @@
+package graphql.analysis;
+
+import graphql.PublicApi;
+import graphql.language.FragmentDefinition;
+import graphql.language.Node;
+import graphql.util.TraverserContext;
+
+@PublicApi
+public interface QueryVisitorFragmentEnvironment {
```

Comment:
same as above: `QueryVisitorFragmentDefitintionEnvironment` is more accurate and more future proof.